From: Andrew Morton <akpm@linux-foundation.org>
Date: Tue, 28 Jan 2025 20:13:57 -0800
Subject: [PATCH] 
 mm-vmscan-extract-calculated-pressure-balance-as-a-function-fix

80-col wrapping

Cc: Hao Zhang <zhanghao1@kylinos.cn>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 mm/vmscan.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mm/vmscan.c b/mm/vmscan.c
index 427d4f360165..bc1826020159 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -2400,8 +2400,8 @@ static void prepare_scan_control(pg_data_t *pgdat, struct scan_control *sc)
 	}
 }
 
-static inline void calculate_pressure_balance(struct scan_control *sc, int swappiness,
-			u64 *fraction, u64 *denominator)
+static inline void calculate_pressure_balance(struct scan_control *sc,
+			int swappiness, u64 *fraction, u64 *denominator)
 {
 	unsigned long anon_cost, file_cost, total_cost;
 	unsigned long ap, fp;
-- 
2.39.5

